### DISCUSSION TITLE
Make a version for Android

### DISCUSSION THREAD

**Discussion by Zac99LOL** [spacedriveapp/spacedrive#1759] [Feature Request]
Please

**Comment by utkubakir** [comment]
We are working on the mobile client :)

**Comment by awidjaja** [comment]
@utkubakir When it the target release date? Tauri 2.0 supports mobile already, are you going to leverage it for the Spacedrive mobile version? It's unfortunately using react native in the readme file.

**Reply by HeavenVolkoff** [reply]
Hi @awidjaja, the mobile app will first launch in a private beta for members of our Discord community with the 0.5.0 release. After we address any bugs identified during the private beta, a public beta will follow shortly. Regarding Tauri mobile, we currently don’t have plans to switch to it, so the mobile app will continue using React Native for the time being. However, most of the app’s logic is shared across desktop, web, and mobile and is implemented in our [`core`](https://github.com/spacedriveapp/spacedrive/tree/main/core) Rust crate. The React Native part is mainly responsible for building the UI and bridging our Rust logic with the native APIs of Android and iOS.
